brickstrap-unshare-harder.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
I need to think about this part some more. Can you make this as a separate pull request?

Ubuntu has `unprivileged_userns_clone` enabled by default, so I think this is only going to be a problem on Debian (and non-Ubuntu relatives). And in Debian, I think it is more likely that root has a password and less likely that a user has sudo setup.